packages/material-ui-private-theming/README.md

```diff
@@ -0,0 +1,3 @@
+# @material-ui/private-theming
+
+Theming utilities to be used in both `@material-ui/styles` & `@material-ui/core`.
```

Comment:
```suggestion
The React theme context to be shared between `@material-ui/styles` and `@material-ui/core`.
```

Reply:
I'll update the `package.json` accordingly 👍